tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchCMISTests.java

```diff
@@ -104,6 +104,22 @@ public void basicCMISQuery()
         searchQueryService.expectSomeResultsFromQuery(query, user1);
     }
 
+    @TestRail (description = "Verify that we can return a list of fields from a document via a CMIS query against Elasticsearch.", section = TestGroup.SEARCH, executionType = ExecutionType.REGRESSION)
+    @Test (groups = TestGroup.SEARCH)
+    public void returnFieldListFromDoc()
+    {
+        SearchRequest query = req("cmis", "SELECT cmis:name, cmis:objectId, cmis:lastModifiedBy, cmis:creationDate, cmis:contentStreamFileName FROM cmis:document WHERE CONTAINS('*')");
```

Comment:
Does the list of field name affect the results ?

If yes, can we check ?

If no, then i assume there must be a negative test - eg. fields that don't exist and cause semantic error ?

Reply:
OK, but do the response differ with different valid fields ?

Reply:
We tried that with @gevagel in enterprise repo and seems that it always returns the same result from the SELECT query regardless of the requested field/fields so the decision was the wait for @tpage-alfresco to confirm that SELECT field is supported

Reply:
FYI: I've tried CMIS Workbench client against a Repo with Solr (SSv2), eg.

`SELECT cmis:name FROM cmis:document WHERE CONTAINS('email')`

I notice client only shows one column (compared to SELECT * which shows multiple columns). I have not analyzed network traffic to see if this is done client-side or server-side (when querying via Solr)

Reply:
Can you try it against Elasticsearch?

Reply:
Unfortunately not - see: https://alfresco.atlassian.net/browse/ACS-2860